lib/handlers/get.js

```diff
@@ -136,7 +135,9 @@ async function handler (req, res, next) {
 
 async function globHandler (req, res, next) {
   const ldp = req.app.locals.ldp
-  const { path: filename } = await ldp.resourceMapper.mapUrlToFile({ url: req })
+  // TODO: This is a hack, as globbing and resource mapping in combination is complex
+  // TODO: Proper support for this is not implemented, as globbing support will probably be removed in the future.
```

Comment:
Perhaps explain what the hack does and how it works?

Nit: globbing support _might_ be removed at some point in the future.